src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoUser } from '../../types/TodoUser';
+import { TodoInfo } from '../TodoInfo';
+
+interface Props {
+  todos: TodoUser[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo: TodoUser, index) => (
+        <TodoInfo todo={todo} key={index} />
```

Comment:
[CODE STYLE] - Avoid using the index as a key in a map function. If the order of items changes, this can lead to bugs in the UI. It's better to use unique and stable identifiers. In this case, you can use 'todo.id' as a key if 'id' is a unique field in your 'todo' objects.